test/fetch.test.js

```diff
@@ -1,3 +1,36 @@
 describe('fetch', function () {
-  it('should fetch')
+  it('should fetch from a URL', function (done) {
+    var Test = this
+    // Don't test fetch for Node
+    try {
+      fetch // eslint-disable-line
+    } catch (e) {
+      return this.skip()
```

Comment:
Skipping the mocha test because the `fetch` polyfill for Node is not actually compatible with the browser `fetch`. We would need to fork the polyfill or have different `fetch` behaviors depending on node/browser.